dali/pipeline/operators/crop/bbox_crop.h

```diff
@@ -164,42 +170,40 @@ class RandomBBoxCrop : public Operator<Backend> {
     return std::make_pair(candidate_boxes, candidate_labels);
   }
 
-  std::tuple<Crop, BoundingBoxes, std::vector<int>> FindProspectiveCrop(
+  std::tuple<Crop, BoundingBoxes, std::vector<int>, bool> FindProspectiveCrop(
```

Comment:
Really, this deserves a proper structure, e.g.
```
struct ProspectiveCrop {
    bool success;
    Crop crop;
    BoundingBoxes boxes;
    std::vector<int> labels;
};
```

Reply:
Done